.readthedocs.yml

```diff
@@ -1,23 +1,24 @@
-# .readthedocs.yml
-# Read the Docs configuration file
-# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
-
-# Required
 version: 2
 
-# Build documentation in the docs/ directory with Sphinx
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+
 sphinx:
-   configuration: docs/source/conf.py
+  builder: html
```

Comment:
i wonder if this conflicts with the `formats` below?

Reply:
it is same as we are used before ,  maybe we can change the formats to `formats: all` to access all the options .
or we can change the sphinx builder to `dirhtml` for simpler URLs .
what do you think ?